microsite/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0",
+  "version": "0.1.1-alpha.19",
```

Comment:
will this have any particular effect? like, refusing to deploy because it's a downgrade or something :)

Reply:
I don't think so. It's a private package and won't be published.